one-trust-notices/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "one-trust-notices",
+  "version": "1.0.0",
+  "description": "",
+  "type": "module",
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "rm -rf dist && tsc",
+    "start": "node dist",
+    "test": "echo \"Error: no test specified\""
+  },
+  
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/html2json": "^1.0.1",
+    "@types/lodash": "^4.14.195",
+    "@types/node": "^20.3.1",
+    "@typescript-eslint/eslint-plugin": "^5.60.0",
+    "@typescript-eslint/parser": "^5.60.0",
+    "eslint": "^8.43.0",
+    "typescript": "^5.1.3"
+  },
+  "dependencies": {
+    "@aws-sdk/client-dynamodb": "^3.354.0",
```

Comment:
generally we prefer fixing the dependency version to avoid the case:
1. develop with dep version 1.0.1
2. release and test in dev
3. dep version 1.1.0 is released
4. we create a tag for our service and a new build runs, downloading the dep version 1.1.0
5. release in uat & prod with a dep we did not test

Not sure if there are other strong opinions about this

Reply:
Ok in that case it should not happen during the deployment. As far as I got it still adds a layer of unpredictability during the development, but I admit I do not have much experience with the tool, so I'll leave it to you ✅